src/modules/transaction/components/Explorer/ExplorerTransactions.js

```diff
@@ -32,7 +35,7 @@ const Transactions = ({ activeToken, address }) => {
   return (
     <Box main className={`${styles.wrapper} transactions-box`}>
       <BoxHeader>
-        <FilterDropdown filters={filters} applyFilters={applyFilters} />
+        <StickyHeader filters={<FilterDropdown filters={filters} applyFilters={applyFilters} />} />
```

Comment:
This seems to have broken the UI and it still dont seem to be sticky 
<img width="851" alt="image" src="https://user-images.githubusercontent.com/8784876/205941300-230d869a-8c0b-494c-b8d1-c1ee75334c35.png">

Reply:
For context, 
The screenshoot is the table on this page http://0.0.0.0:8080/#/wallet

And the fix you made did fix a bug on this page http://0.0.0.0:8080/#/transactions